src/main/java/proto/Pose.proto

```diff
@@ -0,0 +1,20 @@
+/*
+ * This proto format is used along with the delimited methods to
+ * store message size for easy decoding/encoding.
+ */
+syntax = "proto3";
+
+package formats;
+option java_package = "proto";
+option java_outer_classname = "SensorData";
+
+message Pose{
+  // Position is in WGS84 latitude, longitude, altitude.
+  double gps_timestamp = 1; // Time in seconds.
+  double lat = 2;  // Degrees in [-90, 90].
+  double lng = 3;  // Degrees in [-180,180].
+  double alt = 4; // Meters
```

Comment:
"Meters above WGS84"? (Or a different specification, if it's not WGS84.)

Reply:
I was a bit confused on the units myself, Jie mentioned some of the units in a previous comment, but I don't think I completely understood what they were all suppose to be?

Reply:
Yes it is "Meters over WGS84 datum"